src/main/java/com/undertheriver/sgsg/foler/domain/FolderColor.java

```diff
@@ -1,5 +1,21 @@
 package com.undertheriver.sgsg.foler.domain;
 
 public enum FolderColor {
-	BLACK, WHITE
+	BLUE("#2DA5D7"),
+	GREEN("#00A09B"),
+	ORANGE("#F99A42"),
+	PURPLE("#9F69DB"),
+	RED("#E64632");
+
+	String colorCode;
+
+	FolderColor(String colorCode) {
+		this.colorCode = colorCode;
+	}
```

Comment:
```suggestion
	private final String colorCode;

	public FolderColor(String colorCode) {
		this.colorCode = colorCode;
	}
```
enum도 멤버의대한 접근지정자는 숨기고, 생성자의 접근지정자를 정해주는게 좋겠네요

Reply:
그렇군요 ENUM 쪽 코드 반영하겠습니다

색상 API 같은 경우에 저도 처음에는 

1. 접속한 유저를 조회
2. 색상을 리턴

으로 구현을 했다가 지금처럼 프론트에서 입력 받도록 바꿔놨었는데 바꿀 당시에는 색상을 리턴 할 때 마다 유저를 조회하는 쿼리를 또 날리는 게 싫었었나봅니다..

근데 지금 메모 생성을 구현하다보니 폴더 생성을 지금과 같은 로직으로 하니까 API 로직의 통일성이 깨지는 부분이 발생하기도 하고, 그런 통일성이 깨지는 이유가 프론트에 의존적이라서 그런 것 같아서 홍빈님께서 말씀하신대로 서버에서 조회 후 리턴하는 방식이 더 나은 것 같습니다

작업 후 푸시하겠습니다~

Reply:
넵 의견반영 감사합니다!
쿼리가 안가는게 좋겠지만, 무거운쿼리는 아니라 괜찮을 것 같습니다!